script/write_config.py

```diff
@@ -76,6 +77,21 @@ def escape(s: str) -> str:
             s[i] = "\\s"
     return "".join(s)
 
+# Some values need to be "transformed", such as being converted from hex to decimal
+def transformValues(dOrig):
+    d = dOrig
```

Comment:
Not sure if this is needed, since this won't copy anything, it will just be a pointer to `dOrig`. You can use `dOrig.copy()` if this is wanted (which I think makes sense in this case).